src/ROUTES.ts

```diff
@@ -612,6 +612,18 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/taxes/new',
         getRoute: (policyID: string) => `settings/workspaces/${policyID}/taxes/new` as const,
     },
+    WORKSPACE_TAXES_EDIT: {
+        route: 'settings/workspaces/:policyID/tax/:taxID',
+        getRoute: (policyID: string, taxID: string) => `settings/workspaces/${policyID}/tax/${encodeURI(taxID)}` as const,
+    },
+    WORKSPACE_TAXES_NAME: {
+        route: 'settings/workspaces/:policyID/tax/:taxID/name',
+        getRoute: (policyID: string, taxID: string) => `settings/workspaces/${policyID}/tax/${encodeURI(taxID)}/name` as const,
+    },
+    WORKSPACE_TAXES_VALUE: {
+        route: 'settings/workspaces/:policyID/tax/:taxID/value',
+        getRoute: (policyID: string, taxID: string) => `settings/workspaces/${policyID}/tax/${encodeURI(taxID)}/value` as const,
+    },
```

Comment:
NAB the url changes when navigating from Edit to Name/Value pages. We can address this as a follow up though

https://github.com/Expensify/App/assets/22219519/22f8124a-0903-439e-abe0-a3b4aec32e09